packages/sdk/src/document/document.ts

```diff
@@ -589,6 +617,17 @@ export class Document<T, P extends Indexable = Indexable> {
    */
   private isUpdating: boolean;
 
+  /**
+   * `broadcastEventHandlers` is a map of broadcast event handlers.
+   * The key is the topic of the broadcast event, and the value is the handler.
+   */
+  private broadcastEventHandlers: Map<
+    string,
+    DocEventCallbackMap<P>['broadcast']
+  >;
```

Comment:
Thank you for pointing this out. You were absolutely right—there was no need to add `broadcastEventHandlers`. I've referenced the GO-SDK, which includes `broadcastEventHandlers`, leading to my confusion. I've since removed it and confirmed that everything works correctly.